frontend/backend/src/model.js

```diff
@@ -125,6 +127,8 @@ const model = {
                 C_dist : 0.0,
                 Yoke_l : 0.0,
                 core1 : 0.0, //to be changed later
+                netCrossSection: 0.0,
+                stackingFactor : 0.0,
             }
         }
     }
```

Comment:
You cannot calculate exponent using ^ in javascript... u have to use Math.pow(equation , 2) for square. Please change that.
Also i just realised that cells J14 , J 16 ,J 18 and J 20 are not implemented. create new parameters in model.others called L_ID_D , L_OD_D  ,  H_ID_D , H_OD_D respectfully and write 4 more functions to calculate these..

Reply:
done